src/brsTypes/components/Poster.ts

```diff
@@ -0,0 +1,32 @@
+import { FieldModel } from "./RoSGNode";
+import { AAMember } from "./RoAssociativeArray";
+import { Group } from "./Group";
+
+export class Poster extends Group {
+    readonly defaultFields: FieldModel[] = [
+        { name: "uri", type: "string" },
+        { name: "width", type: "float", value: "0.0" },
+        { name: "height", type: "float", value: "0.0" },
+        { name: "loadSync", type: "boolean", value: "false" },
+        { name: "loadWidth", type: "float", value: "0.0" },
+        { name: "loadHeight", type: "float", value: "0.0" },
+        { name: "loadDisplayMode", type: "string", value: "noScale" },
+        { name: "loadStatus", type: "string", value: "noScale" },
+        { name: "bitmapWidth", type: "float", value: "0.0" },
+        { name: "bitmapHeight", type: "float", value: "0.0" },
+        { name: "bitmapMargins", type: "assocarray" },
+        { name: "blendColor", type: "string", value: "0xFFFFFFFF" },
```

Comment:
So the type for this is actually `color` in the docs, which I thought was an alias for `string`. But upon closer inspection in RBI, it appears to be an alias for `integer`. Also, RBI does seem to do some "is this a valid color hex"-checking because, for example, setting it to `10` didn't work, but setting it to `-10` did.

Reply:
That's pretty strange.  I've definitely seen odd behavior with colors in RBI but wasn't sure if it was simply a type-casting issue.  Being able to provide a negative value _feels_ like an integer overflow, but `0xFFFFFFFF` fits evenly into a 32-bit unsigned integer.  Using a 16-bit integer would be strange though, since that fits only up to 65,535 (and 0xFFFFFF without the opacity portion converts to 16,777,215).

🤔 I have no idea how RBI handles this then.

Reply:
Filed https://github.com/sjbarag/brs/issues/417 to investigate and fix this at a later date!